ext/test/w3c_tracecontext_test/main.cc

```diff
@@ -63,6 +65,22 @@ class TextMapCarrierTest : public context::propagation::TextMapCarrier
     headers_[std::string(key)] = std::string(value);
   }
 
+  static bool equalsIgnoreCase(const std::string &str1, const std::string &str2)
```

Comment:
As written, this is a static method in class TextMapCarrierTest.

Please move this code outside the class, so it becomes just a global static function.